openstack/octavia/values.yaml

```diff
@@ -140,14 +140,16 @@ mysql_metrics:
       help: Seconds since last non-pending state of LB
       query: |
         (SELECT
-          UNIX_TIMESTAMP(now()) - UNIX_TIMESTAMP(coalesce(updated_at, created_at)) AS elapsed_time,
-          name AS loadbalancer_name,
-          provisioning_status AS current_status,
-          id AS loadbalancer_id,
-          server_group_id AS octavia_host
-        FROM load_balancer
-        WHERE provisioning_status LIKE "PENDING_%")
-        UNION (select '0', '', '', '', '');
+          UNIX_TIMESTAMP(now()) - UNIX_TIMESTAMP(coalesce(lb.updated_at, lb.created_at)) AS elapsed_time,
+          lb.name AS loadbalancer_name,
+          lb.provisioning_status AS loadbalancer_status,
+          lb.id AS loadbalancer_id,
+          lb.server_group_id AS loadbalancer_host,
```

Comment:
If you rename the variable `octavia_host`, you need to adapt the labels to be exported by the exporter (see line #137), also this change would not be consistent with line 123 of octavia_loadbalancers_count_gauge metric, which still uses `octavia_host` label.
As last, if you change labels, please also change the octavia alerts in the same PR since else you would introduce a window in which the alerts would be broken.

All in all, I would recommend to stay with octavia_host, since it's purely cosmetic.

Regards the network_id, I am fine, but it needs to be added to the labels.

Reply:
Thanks, I totally missed the labels..